Makefile

```diff
@@ -9,3 +9,12 @@ node3:
 
 phoenix:
 	MIX_ENV=dev iex --name phoenix@127.0.0.1 -S mix phx.server
+
+node1_limited:
+	MIX_ENV=dev1 iex --erl "+MMsco true +MMscs 2500" --name dev1@127.0.0.1 -S mix run
```

Comment:
extract these args to a variable, or extract the entire iex command to a separate target o variable, and set the parts that change as variables

Reply:
Do you mean something like this?
```makefile
MEMORY_LIMIT := "+MMsco true +MMscs 2500"

node1_limited:
	MIX_ENV=dev1 iex --erl $(MEMORY_LIMIT) --name dev1@127.0.0.1 -S mix run
node2_limited:
	MIX_ENV=dev2 iex --erl $(MEMORY_LIMIT) --name dev2@127.0.0.1 -S mix run
node3_limited:
	MIX_ENV=dev3 iex --erl $(MEMORY_LIMIT) --name dev3@127.0.0.1 -S mix run
phoenix_limited:
	MIX_ENV=dev iex --erl $(MEMORY_LIMIT) --name phoenix@127.0.0.1 -S mix phx.server
```

Reply:
yeah but more like VM_ARGS or ERL_ARGS